pkg/config/config.go

```diff
@@ -134,31 +89,44 @@ func InitConfig() error {
 		return err
 	}
 	configFile2 := path.Join(configFilePath2, ".atmos", g.ConfigFileName)
-	err = processConfigFile(configFile2, v)
+	found, err = processConfigFile(configFile2, v)
 	if err != nil {
 		return err
 	}
+	if found {
+		configFound = true
+	}
 
 	// Process config in the current dir
 	configFilePath3, err := os.Getwd()
 	if err != nil {
 		return err
 	}
 	configFile3 := path.Join(configFilePath3, g.ConfigFileName)
-	err = processConfigFile(configFile3, v)
+	found, err = processConfigFile(configFile3, v)
 	if err != nil {
 		return err
 	}
+	if found {
+		configFound = true
+	}
 
 	// Process config from the path in ENV var `ATMOS_CLI_CONFIG_PATH`
 	configFilePath4 := os.Getenv("ATMOS_CLI_CONFIG_PATH")
 	if len(configFilePath4) > 0 {
 		u.PrintInfoVerbose(fmt.Sprintf("Found ENV var ATMOS_CLI_CONFIG_PATH=%s", configFilePath4))
 		configFile4 := path.Join(configFilePath4, g.ConfigFileName)
-		err = processConfigFile(configFile4, v)
+		found, err = processConfigFile(configFile4, v)
 		if err != nil {
 			return err
 		}
+		if found {
+			configFound = true
+		}
+	}
+
+	if !configFound {
+		return errors.New("\nno CLI config files 'atmos.yaml' found in any of the searched paths: system dir, home dir, current dir, ENV vars")
```

Comment:
Can we link to one in the repo in the output ?

```suggestion
		return errors.New("\nno CLI config files 'atmos.yaml' found in any of the searched paths: system dir, home dir, current dir, ENV vars\n\nwget https://raw.githubusercontent.com/cloudposse/atmos/master/examples/complete/atmos.yaml")
```